### PR TITLE
Add documentation for Alluxio

### PR DESCRIPTION
**Pull request by mattsfuller**
Add new connector documentation for Alluxio Data Orchestrator.

### CODE REVIEW

**Comment by mattsfuller** [comment]
We added this into the Starburst Presto documentation. Would like to put int the PrestoSQL documentation.

File: presto-docs/src/main/sphinx/connector/alluxio.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+--------------------
+- `What is Alluxio? <https://www.alluxio.io/?utm_source=starburst&utm_medium=prestodocs>`_
+- `Presto with Alluxio <https://www.alluxio.io/presto/?utm_source=starburst&utm_medium=prestodocs>`_
+- `Performance tuning tips for Presto with Alluxio <https://www.alluxio.io/blog/top-5-performance-tuning-tips-for-running-presto-on-alluxio-1/?utm_source=starburst&utm_medium=prestodocs>`_
+
+
+
+
+
+
```
Comment (kokosing) [inline presto-docs/src/main/sphinx/connector/alluxio.rst]: extra empty lines

Comment (mattsfuller) [reply presto-docs/src/main/sphinx/connector/alluxio.rst]: removed

**Comment by mattsfuller** [comment]
updated PR.

**Comment by apc999** [comment]
@mattsfuller not sure if the check failure is related to this Pull request. Any admins to help check?

**Comment by apc999** [comment]
Any admin to comment? thanks

**Comment by electrum** [comment]
Let’s merge #1594 first, then we don’t need the instructions about coping the JAR.

**Comment by ebyhr** [comment]
@mattsfuller Could you remove the section of copying JAR? Related PR was merged.

**Comment by mattsfuller** [comment]
Updates to remove that section.

**Comment by electrum** [comment]
Alluxio isn’t a separate connector. We should instead add an Alluxio section to the Hive connector documentation. I can try reworking this in a new PR, unless @apc999 wants to try?

**Comment by apc999** [comment]
@electrum I created a separate PR #1928 to add Alluxio to the Hive connector doc. Feel free to work with me on the other PR

**Comment by electrum** [comment]
Replaced by https://github.com/prestosql/presto/pull/1928

**Pull request closed without merging**